src/main/java/at/hannibal2/skyhanni/features/garden/visitor/VisitorListener.kt

```diff
@@ -171,7 +188,32 @@ class VisitorListener {
         if (!GardenAPI.onBarnPlot) return
         if (!VisitorAPI.inInventory) return
         val visitor = VisitorAPI.getVisitor(lastClickedNpc) ?: return
+        val isRefuseSlot = event.itemStack.toString() == "1xtile.clayHardenedStained@14"
+        val isAcceptSlot = event.itemStack.toString() == "1xtile.clayHardenedStained@13"
         VisitorToolTipEvent(visitor, event.itemStack, event.toolTip).postAndCatch()
+
+        if (config.rewardWarning.bypassKey.isKeyHeld()) return
```

Comment:
Move the whole code starting here in a new class that listens to the VisitorToolTipEvent